prow/config/config.go

```diff
@@ -119,6 +119,18 @@ type OwnersDirBlacklist struct {
 	Default []string `json:"default"`
 }
 
+// DirBlacklist returns directories which are used to ignore when
+// searching for OWNERS{,_ALIAS} files in a repo.
+func (ownersDirBlacklist OwnersDirBlacklist) DirBlacklist(org, repo string) []string {
+	if bl, ok := ownersDirBlacklist.Repos[org]; ok {
+		ownersDirBlacklist.Default = append(ownersDirBlacklist.Default, bl...)
```

Comment:
There is no pointer in ownersDirBlacklist，It won't update the original the ownersDirBlacklist.Default value。
```go
func (ownersDirBlacklist OwnersDirBlacklist) DirBlacklist(org, repo string) []string {
	if bl, ok := ownersDirBlacklist.Repos[org]; ok {
		ownersDirBlacklist.Default = append(ownersDirBlacklist.Default, bl...)
```
This may be ambiguous， I agree with your suggestion to modify here by adding new variables